### ISSUE TITLE
Add nuget packaging

### ISSUE THREAD

**Issue by mwpowellhtx** [jquery/PEP#288]
First of all, this is a question. Are there plans for NuGet packaging?

Assuming there is not, would you mind elaborating on the jQuery or other dependencies required?

I'm considering adding it, but don't want to duplicate effort if it's already done.

Thanks!

**Comment by arschmitz** [comment]
As for nuget I can't speak for other team members but I would say that would be fairly unlikely. As for dependencies there are none. PEP is a jQuery Foundation project but has no dependency on jQuery

**Comment by scottgonzalez** [comment]
I would like to see npm be the de facto package manager for JavaScript projects, so that is the single package manager I would like to officially support. I prefer having community members manage publishing to other package managers. I think this is helpful if they don't have an easy way to support npm.

If you'd like to maintain the NuGet package for PEP, that would be great. I can notify you every time we release a new version so that you know to update the NuGet package.

As @arschmitz said, PEP has no dependencies. The [browser support](https://github.com/jquery/PEP#browser-compatibility) is documented in the README.

I'm going to close this issue just to keep our bug tracker clean, but feel free to continue commenting if you'd like to own the NuGet package.

Thanks!

**Comment by mwpowellhtx** [comment]
@arschmitz Thanks for the feedback.

@scottgonzalez Thanks, will do. I'll have a look around the repo, fork/clone, and and see what I can do. I'm interested to use it for drag and drop in a BabylonJS 3D browser app and would like to see some packages I can depend on. Any dependencies I should know about? Specifically to do with jQuery, per se?

Thanks!

**Comment by mwpowellhtx** [comment]
Need some level of explanation.

You have a Travis CI running which outputs the distribution modules?

What I want to do is connect a step in the CI that builds the nuget package and potentially uploads it to nuget.org.

If there's no other interest in nuget, I will use my own credentials if it's all the same.

Thoughts?

**Comment by mwpowellhtx** [comment]
Looks like we would need [grunt-nuget](http://www.npmjs.com/package/grunt-nuget), for starters.

**Comment by scottgonzalez** [comment]
> Any dependencies I should know about?

No. This is the third time in this thread that we've told you there are no dependencies.

> You have a Travis CI running which outputs the distribution modules?

Not quite. CI does do a build, but we never distribute from CI. We have a completely separate release script for distribution.

> What I want to do is connect a step in the CI that builds the nuget package and potentially uploads it to nuget.org.

Putting this in our CI or release script would mean that we're owning it. As I said, I want nothing related to NuGet, or any other package manager other than npm, in our processes. We can notify you every time we release a new version and you would be responsible for the work of publishing to NuGet.

**Comment by mwpowellhtx** [comment]
I don't care about NPM. I am using PEP in a context where NuGet is the de facto standard. If you'd like to work in parallel to establish delivery paths into either, I'm open to that.

**Comment by mwpowellhtx** [comment]
Again will ask, there are any dependencies I should be concerned about? For instance, the name suggests "jQuery", or is that "merely" by foundation association? Thank you...

**Comment by dmethvin** [comment]
The package.json lists no dependencies, and the README.md doesn't say there are any, it just says you *can* use it with jQuery. If you use it based on the README.md do you find there are dependencies not mentioned?

**Comment by mwpowellhtx** [comment]
I always like to ask. Thank you...

**Comment by mwpowellhtx** [comment]
@scottgonzalez I will have a couple of changes to PR establishing NuGet if you would please. I'll keep them as incremental as I can. Thank you...

**Comment by scottgonzalez** [comment]
@mwpowellhtx I'm sorry, at this point I will have to ask you to just stop. You're clearly ignoring everything we say and this is not productive.

**Comment by mwpowellhtx** [comment]
You can ask.

**Comment by arschmitz** [comment]
I'm going to lock this issue this is not productive
